lib/asset_host/asset.rb

```diff
@@ -105,7 +110,11 @@ def connection
           Faraday.new(
             :url    => "#{config.protocol || 'https'}://#{config.server}",
             :params => { auth_token: config.token },
-            :ssl => {verify: false}
+            :ssl => {verify: false},
+            :request => {
+                :open_timeout => 2,
```

Comment:
I had to look this up, but I think it should be `:open_timeout => 1` and `:timeout => 2`

If it takes longer than 1 second to open a connection to assethost, or more than 2 seconds to receive the first block of a response, then assume Assethost is slow.

https://stackoverflow.com/questions/10322283/what-is-timeout-and-open-timeout-in-faraday

Reply:
Thanks @nakedsushi! That makes sense and should loosen SCPRv4's reliance on AssetHost more. I'll add that to the next commit in a bit